x-pack/plugins/fleet/server/types/models/package_policy.ts

```diff
@@ -150,4 +151,24 @@ export const PackagePolicySchema = schema.object({
   ...PackagePolicyBaseSchema,
   id: schema.string(),
   version: schema.maybe(schema.string()),
+  revision: schema.number(),
+  updated_at: schema.string(),
+  updated_by: schema.string(),
+  created_at: schema.string(),
+  created_by: schema.string(),
+  elasticsearch: schema.maybe(
+    schema.object({
+      privileges: schema.maybe(
+        schema.object({
+          cluster: schema.maybe(schema.arrayOf(schema.string())),
+        })
+      ),
+    })
+  ),
+  inputs: schema.arrayOf(
+    schema.object({
+      ...PackagePolicyInputsSchema,
+      compiled_input: schema.maybe(schema.any()),
+    })
+  ),
```

Comment:
Hey @joshdover, I re-request your review according to this change, I tried to use this scheme for validation and found that is not aligned with the PackagePolicy type. 

```typescript
export interface PackagePolicy extends Omit<NewPackagePolicy, 'inputs'> {
  id: string;
  inputs: PackagePolicyInput[];
  version?: string;
  revision: number;
  updated_at: string;
  updated_by: string;
  created_at: string;
  created_by: string;
}```

Reply:
Thanks for the ping, this change makes sense. At first I thought that maybe the `elasticsearch` key should be duplicated / moved to `CreatePackagePolicyProps` as well but that would allow setting it on the API which we explicitly do not support. Everything else LGTM, thanks @CohenIdo

In the future, we may want to try refactoring this to derive the TS types from the config schema using the `TypeOf` helper from `kbn/config-schema`.